src/formats/PDB.cpp

```diff
@@ -236,8 +242,15 @@ bool forward(TextFile& file) {
     while (true) {
         try {
             auto line = file.readline();
+
+            if (line.substr(0, 6) == "ENDMDL" &&
```

Comment:
Maybe you can add a comment here to tell what this condition is doing ?

Reply:
done.